plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -376,30 +376,36 @@ def _find_node(self, fname, config=None):
         results = [self.dbt_manifest.expect(uid) for uid in selected]
 
         if not results:
-            model_name = os.path.splitext(os.path.basename(fname))[0]
-            if DBT_VERSION_TUPLE >= (1, 0):
-                disabled_model = None
-                for key, disabled_model_nodes in self.dbt_manifest.disabled.items():
-                    for disabled_model_node in disabled_model_nodes:
-                        if os.path.abspath(
-                            disabled_model_node.original_file_path
-                        ) == os.path.abspath(fname):
-                            disabled_model = disabled_model_node
-            else:
-                disabled_model = self.dbt_manifest.find_disabled_by_name(
-                    name=model_name
-                )
-            if disabled_model and os.path.abspath(
-                disabled_model.original_file_path
-            ) == os.path.abspath(fname):
```

Comment:
I moved this code into a separate function because it was already pretty complex, and this PR only adds to that.